ReactiveCocoaTests/Swift/SignalSpec.swift

```diff
@@ -1058,6 +1058,27 @@ class SignalSpec: QuickSpec {
 				expect(signal).notTo(beNil())
 			}
 
+			it("should not send values when it's less than interval") {
```

Comment:
"Should not send values when it's less than interval" seems synonymous with "should send values on the given scheduler at no less than the interval", the name of the next example. It's not clear to me if they are testing distinct properties of `throttle`. Would it make sense to combine the test code into a single example?